x-pack/plugins/index_lifecycle_management/public/application/services/policies/default_new_policy.ts

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
As this only exports constants, I would prefer to export them from the "constants" folder in its own file.